lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_handlers.cpp

```diff
@@ -1475,8 +1475,10 @@ void sgw_s8_proc_s11_create_bearer_rsp(
   if (pgw_ni_cbr_proc && (LIST_EMPTY(pgw_ni_cbr_proc->pending_eps_bearers))) {
     pgw_base_proc_t* base_proc1 = LIST_FIRST(sgw_context_p->pending_procedures);
     LIST_REMOVE(base_proc1, entries);
-    free_wrapper((void**)&sgw_context_p->pending_procedures);
-    free_wrapper((void**)&pgw_ni_cbr_proc->pending_eps_bearers);
+    free_cpp_wrapper(
+        reinterpret_cast<void**>(&sgw_context_p->pending_procedures));
+    free_wrapper(
```

Comment:
Shouldn't this be `free_cpp_wrapper`?

Reply:
In this PR, for pending_eps_bearers memory is allocated with calloc. But in next PR I have changed to new()